src/main/java/org/spongepowered/api/profile/property/ProfileProperty.java

```diff
@@ -41,6 +43,13 @@
  */
 public interface ProfileProperty {
 
+    /**
+     * The name of the {@code textures} property.
+     *
+     * This is used with {@link SkinData} to set the skin of a {@link Humanoid}
```

Comment:
`<p>This is used with {@link SkinData} to set the skin of a {@link Humanoid}.</p>`

Reply:
@Aaron1011